packages/editor/src/plugins/jsx-blocks/index.js

```diff
@@ -44,12 +45,14 @@ export default (opts = {}) => ({
   },
   renderNode: (props, editor, next) => {
     const { node } = props
+    if (node.type !== 'jsx-void') return next()
+    const component = node.data.get('type')
 
-    switch (node.type) {
-      case 'youtube':
+    switch (component) {
```

Comment:
This could check for the existence of `components[component]` instead of being hard-coded